client/app/telecom/telephony/billingAccount/orderAlias/geographical/telecom-telephony-billing-account-orderAlias-geographical.html

```diff
@@ -67,7 +67,7 @@
                                data-ng-model="AliasOrderGeographicalCtrl.form.zone"
                                data-ng-change="AliasOrderGeographicalCtrl.changeZone()"
                                data-ng-disabled="AliasOrderGeographicalCtrl.loading.init || AliasOrderGeographicalCtrl.loading.order"
-                               data-uib-typeahead="item as item.askedCity for item in AliasOrderGeographicalCtrl.getGeographicalZone($viewValue)"
+                               data-uib-typeahead="item as item.city for item in AliasOrderGeographicalCtrl.getGeographicalZone($viewValue)"
```

Comment:
How could this mistake reach production ? :/

Reply:
Hmm maybe this prop is not set anymore by the API.